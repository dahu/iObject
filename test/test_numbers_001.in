" test iObject Numbers

1
1 2
ax1
ax1ax
ax 1 ax

