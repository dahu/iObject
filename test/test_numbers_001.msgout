
"test_numbers_001.in" 
"test_numbers_001.in" 8L, 49C
Warning: terminal cannot highlight






"test_numbers_001.out" 
"test_numbers_001.out" [New] 8L, 49C written
