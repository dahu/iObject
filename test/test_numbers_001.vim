source ../plugin/iObject.vim
edit test_numbers_001.in

" change unsigned numbers
1
call search('\d', 'W')
normal cani
normal j
normal canj
normal w
normal cank
call search('\d', 'W')
normal canl
call search('\d', 'W')
normal canm
call search('\d', 'We')
normal cann

write test_numbers_001.out
quit!
