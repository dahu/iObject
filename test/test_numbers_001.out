" test iObject Numbers

i
j k
axl
axmax
ax n ax

